plugins/kubernetes/app/models/kubernetes/resource.rb

```diff
@@ -27,15 +27,19 @@ def patch_replace
           next unless value = @template.dig(*keys)
           update.dig_set keys, value
         end
-        with_patch_header do
+        with_header 'application/json-patch+json' do
```

Comment:
This one is covered by `json_patch` since https://github.com/abonas/kubeclient/pull/390/files

Reply:
I need the `with_header` for apply anyway, so just leaving that ... `+json` does not work for apply for some reason, it only supports `+yaml` :/